Sources/Sentry/Public/SentrySdkInfo.h

```diff
@@ -33,6 +33,9 @@ SENTRY_NO_INIT
 
 - (instancetype)initWithDict:(NSDictionary *)dict;
 
+- (instancetype)initWithDict:(NSDictionary *)dict
+                  orDefaults:(SentrySdkInfo *)info;
```

Comment:
wasn't sure about the naming convention here...

Reply:
I think that's fine.